mkat_fpga_tests/test_cbf.py

```diff
@@ -589,7 +589,7 @@ def get_actual_phases():
                 delays[setup_data['test_source_ind']] = delay
                 delay_coefficients  = ['{},0:0,0'.format(dv) for dv in delays]
                 reply = correlator_fixture.katcp_rct.req.delays(
-                    time.time()+delay, *delay_coefficients)
+                    time.time()+.2, *delay_coefficients)
```

Comment:
Remember, that is the application time. We want that to be as soon as possible, but there is a minimum lead time required by the CBF. Also, this will be affected by epoch-drift between the CMC and the test process. You might be better off getting a clean dump, taking its timestamp, and then setting it one dump + 500 ms in the future if you want this test to be reliable?